.env.example

```diff
@@ -0,0 +1,3 @@
+GOOSE_DRIVER=postgres
```

Comment:
Mind removing this file, don't think we need to include it and probably sufficient to document it either in the README or docs site https://pressly.github.io/goose/

Reply:
extended `README.md` with this example